mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -278,34 +279,45 @@ class Word2VecModel private[ml] (
   @Since("1.4.0")
   def setOutputCol(value: String): this.type = set(outputCol, value)
 
+  private var bcModel: Broadcast[Word2VecModel] = _
```

Comment:
I don't suppose we have a way to clean this up after use - will just have to get GCed?

Reply:
yes. I followed the impl of [CountVectorizer](https://github.com/apache/spark/blob/master/mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala#L302) here.
Since other .ml impls do not use a mutable `var` for a broadcast variable like this, I will remove this var.

Reply:
As to `CountVectorizer`, should we also remove the `var broadcastDict` in it? It looks like that other mllib impls do not use mutable broadcasted variable like that.